bandhuapp/models.py

```diff
@@ -1,10 +1,18 @@
+import hashlib
+import time
+
 from django.db import models
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 from django.utils import timezone
 
 from accounts.models import User
 
+def _createHash():
+    hash = hashlib.sha1()
+    hash.update(str(time.time()).encode('utf-8'))
+    return hash.hexdigest()[0:20]
+
```

Comment:
This can be put inside `UrlData` class itself or in a separate helper file. Keep only model classes at the top-level of models.py file.

Reply:
Okay, I will update this.